src/api/forex/forex.js

```diff
@@ -30,8 +30,31 @@ export default class Forex {
             );
             return await response.json;
         } catch (err) {
-            const error = await determineError(err);
-            throw error;
+            throw await determineError(err);
+        }
+    }
+
+    /**
+     * Use the Forex (FX) rates API to get the indicative foreign exchange rates that Checkout.com
+     * uses to process payments for card payouts
+     *
+     * @param {Object} body Forex params.
+     * @return {Promise<Object>} A promise to the Forex response.
+     */
+    async getRates(body) {
+        try {
+            let url = `${this.config.host}/forex/rates`;
+
+            if (body) {
+                const queryString = Object.keys(body)

```

Comment:
Can't we make this block as util class and just call this reference? We don't need to do the refactor for all the classes, but we can just start making more reusable code.